### DISCUSSION TITLE
How can i found my link for prisma data browser

### DISCUSSION THREAD

**Discussion by plastikmind** [prisma/prisma#24614] [Q&A]
### Question

Hi everyone, I've successfully connected my database with Pulse on the Prisma Data Platform. However, I'm unable to find the link to access Prisma Studio online. Could someone guide me on how to locate and use Prisma Studio with my connected database through the Prisma Data Platform?

Thanks in advance!

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by petradonka** [comment]
Hi @plastikmind: great to hear you're trying out Pulse!

You can simply run `npx prisma studio` locally to launch Studio. This will connect to the database defined in your schema file.

More info via these links:
- https://www.prisma.io/docs/orm/tools/prisma-studio
- https://www.prisma.io/docs/pulse/getting-started#3-test-your-stream

Let us know if there's anything else we can help with!

**Reply by plastikmind** [reply]
Hi @petradonka 

Thanks for your response!

I understand how to run Prisma Studio locally. I'm specifically looking for a way to share access to Prisma Studio online with other admins. My goal is to enable our team to view and edit the database remotely at any time, without needing to run the Studio locally on their machines.

From my understanding, the Prisma Data Platform includes a feature called the Data Browser, which seems to allow collaborative online access to the database. Is there a way to use this feature, or any other method within the Prisma Data Platform, to achieve this functionality?

Here's what I have set up:

Database connected with Pulse on the Prisma Data Platform
Looking to provide online, collaborative access to the database for other admins
Is there a feature in Prisma Data Browser that supports this, or any other recommended approach?

Thanks in advance!

**Reply by petradonka** [reply]
Hi @plastikmind: thanks for the clarification!

Data Browser was part of a feature set we deprecated last year, so it is not currently part of Platform.

But, we do have some exciting updates coming to Studio, and the workflow you described is one that is important for us that we cover with the right feature set! So keep an eye on future feature announcements regarding Studio. 😉 

In the meantime, you could host Studio yourself, e.g. via https://www.flightcontrol.dev/docs/reference/examples/prisma-studio

Hope that helps!
